examples/plugin/render_plane_manipulator.py

```diff
@@ -28,7 +27,7 @@
     colormap='bop orange',
     blending='additive',
     opacity=0.5,
-    experimental_slicing_plane=plane_parameters
+    plane=plane_parameters
```

Comment:
Does this change mean we have to set a minimum napari version?

Reply:
yeah, to be safe let's go 0.4.16+ - I didn't mean to include these changes... I will revert